patch/src/lib.rs

```diff
@@ -532,6 +532,29 @@ fn create(
     Ok(())
 }
 
+/// Create a human friendly message about git's sync status.
+fn pretty_sync_status(
+    repo: &git::Repository,
+    revision_oid: git::Oid,
+    head: &git::Reference,
+) -> anyhow::Result<String> {
+    let target = head.target();
+    if target == None {
+        return Ok(String::default());
+    }
+    let head_oid = target.unwrap();
```

Comment:
No need to unwrap here if you use `if let`

Reply:
Noted, thanks :)